src/display/api.js

```diff
@@ -957,10 +957,41 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
      */
     getTextContent: function PDFPageProxy_getTextContent(params) {
       params = params || {};
-      return this.transport.messageHandler.sendWithPromise('GetTextContent', {
+      var readableStream = this.transport.messageHandler
+                                           .sendWithStream('GetTextContent', {
         pageIndex: this.pageNumber - 1,
         normalizeWhitespace: (params.normalizeWhitespace === true),
         combineTextItems: (params.disableCombineTextItems !== true),
+      }, {
+        highWaterMark: 5,
```

Comment:
ReadableStream needs to know its "buffer size". I'm thinking to keep it around 100 as a starter, but it can go lower. `TEXT_CONTENT_CHUNK_SIZE` as a name?